attendance/views.py

```diff
@@ -135,10 +135,10 @@ def list(self, request, *args, **kwargs):
                 if date:
                     datetime.strptime(date, '%Y-%m-%d')
                     record = Attendance.objects.filter(check_in__date=date,
-                                                       is_deleted=False).order_by('-check_in__date')
```

Comment:
-check_in__time is no different than -check_in__date. I'll sort the objects on the basis of time, and totally ignoring date. 
-'check_in' is better solution.